bitnami/apache/templates/deployment.yaml

```diff
@@ -58,9 +58,6 @@ spec:
           initialDelaySeconds: 5
           timeoutSeconds: 3
           periodSeconds: 5
```

Comment:
Now that I think of it. How is the user supposed to mount their own web page? Having just an Apache server deployed does not add much value to the chart, does it?

Reply:
I will create a task to investigate it